vnext/Microsoft.ReactNative/Views/TextViewManager.cpp

```diff
@@ -270,6 +270,17 @@ void TextViewManager::RemoveChildAt(const XamlView &parent, int64_t index) {
   return textBlock.Inlines().RemoveAt(static_cast<uint32_t>(index));
 }
 
+void TextViewManager::ReplaceChild(const XamlView &parent, const XamlView &oldChild, const XamlView &newChild) {
+  const auto textBlock{parent.as<xaml::Controls::TextBlock>()};
+  const auto oldInline{oldChild.as<xaml::Documents::Inline>()};
+  const auto newInline{newChild.as<xaml::Documents::Inline>()};
```

Comment:
shouldn't these be TextElements so they can work with Blocks too?

Reply:
We don't use blocks in RNW, but that's a good way to future proof.